### PR TITLE
Replace OCI Java SDK shaded jar with v3 for OCI integration

### PR DESCRIPTION
**Pull request by klustria**

### CODE REVIEW

File: integrations/oci/sdk/cdi/pom.xml [hunk 1]
```diff
@@ -37,25 +37,18 @@
...
-
         <dependency>
             <groupId>com.oracle.oci.sdk</groupId>
-            <artifactId>oci-java-sdk-shaded-full</artifactId>
+            <artifactId>oci-java-sdk-common</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.oracle.oci.sdk</groupId>
+            <artifactId>oci-java-sdk-common-httpclient-jersey3</artifactId>
```
Comment (ljnelson) [inline integrations/oci/sdk/cdi/pom.xml:50]: (I'd add `<scope>compile</scope>` for clarity.)

**Review by ljnelson — APPROVED** [review]
LGTM only one minor comment

File: integrations/oci/sdk/cdi/pom.xml (hunk 1 above)
Comment (klustria) [reply integrations/oci/sdk/cdi/pom.xml:50]: Updated as suggested.

**Comment by klustria** [comment]
Updated problem with oci atp example where pom.xml is referencing v2.5.5 rather than v3.0.3.

**Review by ljnelson — DISMISSED** [review]

**Comment by klustria** [comment]
Tested Monitoring and Logging. Also tested Object Storage Example.

**Review by ljnelson — DISMISSED** [review]

**Comment by klustria** [comment]
Tested OCI vault example and works well.

**Comment by klustria** [comment]
The examples for vault-reactive and objectstorage-reactive also requires jersey3 httpclient dependency to be added.

**Review by arjav-desai — DISMISSED** [review]
LGTM

**Review by barchetta — APPROVED** [review]

**Pull request merged**